Failing to build site locally

Hi,

I followed the instructions in CONTRIBUTING.md and can't build the site locally. I get the following error:

```
 ~/src/ratatui-website main pnpm dev

> ratatui.rs@0.0.1 dev /home/alabhya/src/ratatui-website
> astro dev

15:11:18 [types] Generated 1ms
15:11:19 [content] Syncing content
15:11:19 [content] Astro config changed
15:11:19 [content] Clearing content store
15:11:28 [content] Synced content

 astro  v5.17.3 ready in 11501 ms

┃ Local    http://localhost:4321/
┃ Network  use --host to expose

15:11:28 watching for file changes...
15:11:36 [ERROR] Cannot find module 'shiki' imported from '/home/alabhya/src/ratatui-website/src/components/ScrambleCode.astro'
  Stack trace:
    at /home/alabhya/src/ratatui-website/src/content/docs/index.mdx?astroPropagatedAssets
    [...] See full stack trace in the browser, or rerun with --verbose.
```

I get error after running `pnpm approve-builds` and `pnpm install` along with the other instructions. 

## Platform
Node: v22.22.0
Firefox: 147.0.3
OS: Fedora Linux 42

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -7,31 +7,36 @@ This repo contains the Astro/Starlight site plus a Rust workspace with the code 
 
 ## Setup
 
-- Node 24.x (lts), pnpm (recommended) or npm, and Git LFS.
+- Node 24.x (lts), npm, and Git LFS.
 - Rust (MSRV 1.86) (only if you change anything in [code/](code/)).
 
 ```sh
 git clone https://github.com/ratatui/ratatui-website
 cd ratatui-website
-git lfs install && git lfs pull
-pnpm install && pnpm dev
-# or: npm install && npm run dev
+
+git lfs install
+git lfs pull
+
+npm install
+npm run dev
 ```
 
-> [!NOTE] `npm install` (and `pnpm install`) triggers Playwright's browser download; you can skip
-> the heavy Chromium bundle by hitting Ctrl+C once that download starts if you do not plan to run
-> the end-to-end tests.
+Using a different package manager may result in errors. pnpm, for example, is known not to work because of it's dependency isolation.
+
+> [!NOTE]
+> `npm install` triggers Playwright's browser download; you can skip the heavy Chromium
+> bundle by hitting Ctrl+C once that download starts if you do not plan to run the end-to-end tests.
 
 ## Commands
 
 - Website (Astro/Starlight):
 
-  - `pnpm dev` or `npm run dev` for viewing locally
-  - Format: `pnpm format` / `npm run format -- --write` (write) or `npm run format -- --check` (note
+  - `npm run dev` for viewing locally
+  - Format: `npm run format -- --write` (write) or `npm run format -- --check` (note
     the double `--`)
-  - Type-check: `pnpm astro check` or `npm run astro check`
-  - Build: `pnpm build` or `npm run build` (runs `astro check`)
-  - Tests: `pnpm test` or `npm test`
+  - Type-check: `npm run astro check`
+  - Build: `npm run build` (runs `astro check`)
+  - Tests: `npm test`
 
 - Code examples:
   - All snippets should live in `code/`, included using `{{ #include @code/... }}`

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -21,7 +21,8 @@ npm install
 npm run dev
 ```
 
-Using a different package manager may result in errors. pnpm, for example, is known not to work because of it's dependency isolation.
+> [!NOTE]  
+> Using a different package manager may result in errors. `pnpm`, for example, is known not to work because of it's dependency isolation.
 
 > [!NOTE]
 > `npm install` triggers Playwright's browser download; you can skip the heavy Chromium

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -22,10 +22,10 @@ npm run dev
 ```
 
 > [!NOTE]  
-> Using a different package manager may result in errors. `pnpm`, for example, is known not to work because of it's dependency isolation.
+> Using a different package manager may result in errors. `pnpm`, for example, is known not to work
+> because of it's dependency isolation.
 
-> [!NOTE]
-> `npm install` triggers Playwright's browser download; you can skip the heavy Chromium
+> [!NOTE] > `npm install` triggers Playwright's browser download; you can skip the heavy Chromium
 > bundle by hitting Ctrl+C once that download starts if you do not plan to run the end-to-end tests.
 
 ## Commands
@@ -33,8 +33,8 @@ npm run dev
 - Website (Astro/Starlight):
 
   - `npm run dev` for viewing locally
-  - Format: `npm run format -- --write` (write) or `npm run format -- --check` (note
-    the double `--`)
+  - Format: `npm run format -- --write` (write) or `npm run format -- --check` (note the double
+    `--`)
   - Type-check: `npm run astro check`
   - Build: `npm run build` (runs `astro check`)
   - Tests: `npm test`

--- a/README.md
+++ b/README.md
@@ -13,4 +13,6 @@ To read this content online, visit:
 
 <https://ratatui.rs>
 
-We welcome contributions! Please see [CONTRIBUTING.md](https://github.com/ratatui/ratatui-website/blob/main/CONTRIBUTING.md) to get started.
+We welcome contributions! Please see
+[CONTRIBUTING.md](https://github.com/ratatui/ratatui-website/blob/main/CONTRIBUTING.md) to get
+started.

--- a/README.md
+++ b/README.md
@@ -11,19 +11,6 @@ rich Terminal User Interfaces.
 
 To read this content online, visit:
 
-<https://ratatui.rs/>
+<https://ratatui.rs>
 
-To build and run the site locally:
-
-```shell
-git clone https://github.com/ratatui/ratatui-website/
-cd ratatui-website
-
-git lfs install
-git lfs pull
-
-npm install
-npm run dev
-```
-
-Feel free to make contributions and submit a PR if you'd like to improve the documentation.
+We welcome contributions! Please see [CONTRIBUTING.md](https://github.com/ratatui/ratatui-website/blob/main/CONTRIBUTING.md) to get started.

